ArduPlane/mode_qstabilize.cpp

```diff
@@ -35,6 +35,11 @@ void ModeQStabilize::update()
         plane.nav_roll_cd = roll_input * plane.quadplane.aparm.angle_max;
         plane.nav_pitch_cd = pitch_input * plane.quadplane.aparm.angle_max;
     }
+
+    if (plane.control_mode->mode_number() == Mode::Number::QSTABILIZE) {
```

Comment:
this check is preventing assign_tilt_to_fwd_thr() being called in QHOVER mode. I just tested QHOVER and the new functionality doesn't work with the RC switch
we can either remove the check and remove the call in qloiter, qrtl etc, or add the call into all the relevant modes
I think it is better to have it in one place?